src/adaptors/tcp_adaptor.c

```diff
@@ -982,13 +982,20 @@ static qdr_tcp_connection_t *qdr_tcp_connection_ingress(qd_tcp_listener_t* liste
     tc->bridge->connections_opened +=1;
     UNLOCK(tc->bridge->stats_lock);
 
+    LOCK(tc->activation_lock);
     tc->pn_raw_conn = pn_raw_connection();
     pn_raw_connection_set_context(tc->pn_raw_conn, tc);
     //the following call will cause a PN_RAW_CONNECTION_CONNECTED
     //event on another thread, which is where the rest of the
     //initialisation will happen, through a call to
     //qdr_tcp_connection_ingress_accept
     qd_log(tcp_adaptor->log_source, QD_LOG_INFO, "[C%"PRIu64"] call pn_listener_raw_accept()", tc->conn_id);
```

Comment:
Should this _really_ be an INFO level log?  Seems more lie a DEBUG log to me.  Ditto below...